packages/react/src/themes/tokens/component/tab-tokens.ts

```diff
@@ -22,16 +23,17 @@ export type TabTokenMap = {
 };
 
 export const defaultTabTokens: TabTokenMap = {
-    'tabs-tab-border-bottom-color': 'color-neutral-50',
-    'tabs-tab-button-text-color': 'color-black',
-    'tabs-tab-hover-background-color': 'color-neutral-15',
-    'tabs-tab-left-icon-color': 'color-black',
-    'tabs-tab-right-icon-color': 'color-black',
-    'tabs-tab-global-text-color': 'color-neutral-90',
-    'tabs-tab-global-selected-background-color': 'color-brand-50',
-    'tabs-tab-text-color': 'color-neutral-50',
-    'tabs-tab-selected-background-color': 'color-white',
-    'tabs-tab-selected-border-color': 'color-neutral-50',
+    'tabs-background-color': 'color-neutral-05',
```

Comment:
```suggestion
    'tab-background-color': 'color-neutral-05',
    'tab-global-background-color': 'color-white',
    'tab-border-color': 'color-neutral-15',
    'tab-icon-color': 'color-neutral-65',
    'tab-text-color': 'color-neutral-65',
    
    'tab-indicator-hover-background-color': 'color-neutral-15',
    'tab-hover-text-color': 'color-neutral-65',
    
    'tab-indicator-pressed-background-color': 'color-brand-70',
    'tab-pressed-text-color': 'color-neutral-90',
    
    'tab-indicator-selected-background-color': 'color-brand-50',
    'tab-selected-text-color': 'color-neutral-90',

    'tab-panel-border-color': 'color-neutral-05',
    'tab-panel-background-color': 'color-white',
```

faudrait demander à @maboilard si active s'inscrit à leur language de design ou si ça serait plus pressed...

Reply:
Actuellement, c'est `Active` mais j'ai l'impression que `Pressed` pourrait être confondant si à un certain point, on décide d'appliquer un nouvel état Pressed au clic. Est-ce que `Selected` serait plus approprié?

Reply:
Je crois que dans ce cas ça serait `active` puisque `selected` est l'état après avoir pressed/active

Reply:
Si on essaie d'utiliser les termes déjà dispo en web.

`active` en css = quand tu cliques sur l'élément. i.e., tu peux changer le style d'un élément clickable au moment tu le trigger avec la souris  https://developer.mozilla.org/en-US/docs/Web/CSS/:active

`aria-selected` en a11y = This attribute is used to indicate which elements within single-selection and multiple-selection composite widgets are selected. https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-selected

`aria-pressed` en a11y = This attribute indicates the current "pressed" state of a toggle button. https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-pressed

Je vous laisse juger quel terme utiliser.

Reply:
Selon les docs, "selected" c'est quand tu l'a choisi (le current Tabs).  "pressed" c'est presque la meme chose mais pour les toggle buttons.

J'ai mis "active" pour quand tu presse dessus a cause du nom en CSS.  Cependant j'ai trouve plusieurs DS qui utilise "active" comme l'equivalent de "selected".

active: Bootstrap, Fluent, Semantic-UI, Ants
selected: Material-UI, Carbon, Atlassian, Adobe

La majorite n'ont pas de style pour quand tu presse dessus, ils ont juste un hover, ou alors c'est fait en CSS, alors j'ai pas trouve comment ils nomme ca.

Si on va avec les specs CSS et A11y, moi j'irais pour:

current tab: selected
mouse pressed: active